contracts/HydraStaking/modules/PenalizeableStaking/PenalizeableStaking.sol

```diff
@@ -128,7 +125,7 @@ contract PenalizeableStaking is IPenalizeableStaking, HydraChainConnector, Staki
         }
     }
 
-    function isStakeLeftLow(uint256 accountStake, uint256 unstakeAmount) private view returns (bool) {
+    function _isStakeLeftLow(uint256 accountStake, uint256 unstakeAmount) private view returns (bool) {
```

Comment:
Must be removed (unused)

Reply:
Removed and stateSync is there as "Balance Changed" event